style.css

```diff
@@ -1,4 +1,233 @@
 * {
-  box-sizing: border-box;
-  margin: 0;
+  /*box-sizing: border-box;*/
+  margin: 0 auto;
+  position: relative;
+}
+
+html {
+  background: url(https://s-media-cache-ak0.pinimg.com/736x/05/71/c0/0571c02063ea91a8dca4c75858712609.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  position: relative;
+}
+
+body {
+  background-repeat: no-repeat;
+
+}
+.left{
+  text-align: left;
+  position: fixed;
+  bottom: 10%;
+  left: 5%;
+  width: 19rem;
+  height: 5rem;
+  transform: translate(-8rem, 0rem);
+  transform: skewY(-3deg);
+  overflow: auto;
+}
+
+ .avatar{
+  float: left;
+
+ }
+
+ .hp {
+   margin-top: 1.5rem;
+   font-family: Rockwell;
+   color: white;
+   font-size: .7rem;
+ }
+
+ .hp--500 {
+   font-size: 1.5rem;
+   font-family: Rockwell;
+ }
+
+.health {
+  border-bottom: white 2px solid;
+  border-right: #209fd5 3px solid;
+  transform: skewX(-15deg);
+  overflow: auto;
```

Comment:
If you've set a hard width or height, setting `overflow: auto` will make scroll bars show up when you might not want it.
You can still get the sizing by using `overflow: hidden` and the scrollbars won't show up.